util/internal/integrationtest/mapper/stub/src/main/java/org/hibernate/search/util/impl/integrationtest/mapper/stub/StubMappedIndex.java

```diff
@@ -169,7 +170,10 @@ public IndexWorkspace createWorkspace(StubSession sessionContext) {
 	}
 
 	public IndexWorkspace createWorkspace(String tenantId) {
-		return delegate().createWorkspace( mapping, tenantId );
+		return delegate().createWorkspace(
+				mapping,
+				tenantId == null ? Collections.emptySet() : Collections.singleton( tenantId )
```

Comment:
You're doing this a lot, all over the place. Maybe consider adding a method to `org.hibernate.search.util.common.impl.CollectionHelper`?

Reply:
Yeah, I've thought about some tenant utils, but that didn't make much sense, good point about the collection helper, will do!